ui/components/component-library/checkbox/checkbox.scss

```diff
@@ -1,6 +1,4 @@
 .mm-checkbox {
-  cursor: pointer;
```

Comment:
Will the label still use the pointer when it is present or just the checkbox? I think we may need it in both but haven't tested yet. Waiting for storybook build on PR